packages/react-core/src/demos/CardDemos.md

```diff
@@ -260,3 +253,163 @@ class CardGridDemo extends React.Component {
   }
 }
 ```
+
+### Horizontal split
+
+```js
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  CardFooter,
+  Grid,
+  GridItem,
+  Button
+} from '@patternfly/react-core';
+
+CardHorizontalSplitDemo = () => {
+  return (
+    <Card id="card-demo-horizontal-split-example" isFlat>
+      <Grid md={6} hasGutter>
+        <GridItem style={{minHeight: '200px', backgroundPosition: 'center', backgroundSize: 'cover', backgroundImage: 'url(/assets/images/pfbg_992@2x.jpg)' }}/>
+        <GridItem>
+          <CardTitle>Headline</CardTitle>
+          <CardBody>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse arcu purus, lobortis nec euismod eu, tristique ut sapien. Nullam turpis lectus, aliquet sit amet volutpat eu, semper eget quam. Maecenas in tempus diam. Aenean interdum velit sed massa aliquet, sit amet malesuada nulla hendrerit. Aenean non faucibus odio. Etiam non metus turpis. Praesent sollicitudin elit neque, id ullamcorper nibh faucibus eget.
+          </CardBody>
+          <CardFooter>
+            <Button variant="tertiary">Call to action</Button>
+          </CardFooter>
+        </GridItem>
+      </Grid>
+    </Card>
+  );
+}
+```
+
+### Card details
```

Comment:
It's called "Details card" on the core side - @mcarrano how should this one be named?